src/main/resources/com/microsoftopentechnologies/windowsazurestorage/Messages.properties

```diff
@@ -13,7 +13,9 @@ WAStoragePublisher_uploading=MicrosoftAzureStorage - Uploading files to Microsof
 WAStoragePublisher_nofiles_uploaded=MicrosoftAzureStorage - Failed to find any build artifacts to upload to Azure Storage \
                                     \nVerify the list of files to upload and that the Ant glob syntax is correct
 WAStoragePublisher_files_uploaded_count=MicrosoftAzureStorage - Uploaded/archived file count = {0}
+WAStoragePublisher_files_need_upload_count=MicrosoftAzureStorage - Need uploaded/archived file count = {0}
 WAStoragePublisher_uploaded_err=MicrosoftAzureStorage - Error occurred while uploading to Azure - {0}
+WAStoragePublisher_uploaded_timeout=MicrosoftAzureStorage - Upload artifacts to Azure timeout of {0} {1}
```

Comment:
This is not the best message. A little bit confusing. Might be something like "... fails due to timeout after ..."

Reply:
changed